### ISSUE TITLE
Small error in Climatic > Check Data > Inventory

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#5408]
I used the inventory for rainfall data where there was the shifted year (starting in July).  Could use the Dodoma data for this.
When I use the options below:
![image](https://user-images.githubusercontent.com/11226469/60570436-7f0a0200-9d69-11e9-80fa-0d45c4062d2f.png)

Then the graph is as follows:
![image](https://user-images.githubusercontent.com/11226469/60570516-a791fc00-9d69-11e9-9738-ea04bb17549e.png)

It should shift the day of year in the plot.  Or it could have the same date layout as shown, i.e. always January to December, but then it needs to get the day of year from the date column.

**Comment by Wycklife** [comment]
@africanmathsinitiative/developers I've taken this up

**Comment by dannyparsons** [comment]
Not sure how this is happening because only the date is supplied from the dialog, none of the shifted columns are being passed in.
I think what's easiest is always going from Jan-Dec because then only date is used. We could add an (optional) receiver for year which could then do a shifted y-axis.

**Comment by Wycklife** [comment]
@dannyparsons How do I go about this issue? Do I add receiver as you suggested before?

**Comment by dannyparsons** [comment]
How about a option to specify a starting date: Day + Month, if the day of year plot is selected, instead of having an additional receiver? The only thing we need here is that date, not the whole column.
It can default to 1 Jan, but can be changed. @rdstern what do you think?

**Comment by rdstern** [comment]
That would be great.  But as it uses the date I would not object to always starting on 1 Jan if the additional features take too long.  It is an inventory after all.

There is also an interesting issue at the top of the plot.  That happens a lot and I am not sure what it is.  Perhaps that should be a separate issue?

In the (slightly I hope) longer term we will need to consider what to do about hourly and other within-day data.  I think @shadrackkibet raised this as a separate issue after his KMD work.

**Comment by shadrackkibet** [comment]
Yes, I had an issue with hourly data, especially for wind which I wanted to produce an inventory plot. We need to see how we cope with that. Generally, I think most facilities that we have in the climatic menu cope well with daily data (I might be wrong). We need to also consider sub-daily data.
